dipper/sources/ZFIN.py

```diff
@@ -1247,16 +1246,15 @@ def _process_g2p(self, limit=None):
         with open(raw, 'r', encoding="utf8") as csvfile:
             filereader = csv.reader(csvfile, delimiter='\t', quotechar='\"')
             for row in filereader:
-                line_counter += 1
-
+                line_counter += 1 
+              
```

Comment:
I have been eliminating these counters as I go and using the one builtin and maintained by the ```csv.reader``` .
so later when "line_counter" would be used, instead favor `filereader.line_num`.
To help keep that reasonable I have also been choosing `reader'`  `writer` over 
"strangely_specific_filesystem_input-output_handler"  or "fh" or "f"

Reply:
Sensible, and will do this in ZFIN.py as soon as I can. Appreciate the style tips Tom!